services/notifications/src/simcore_service_notifications/api/rpc/routing.py

```diff
@@ -0,0 +1,23 @@
+from collections.abc import AsyncIterator
+
+from fastapi import FastAPI
+from fastapi_lifespan_manager import State
+from models_library.api_schemas_notifications import NOTIFICATIONS_RPC_NAMESPACE
+from servicelib.rabbitmq import RPCRouter
+
+from ...clients.rabbitmq import get_rabbitmq_rpc_server
+
+ROUTERS: list[RPCRouter] = [
+    # import form various domains and attach here
```

Comment:
typo: `form` --> `from`


minor: since there are no RPC routers I think this whole file should be deleted and only introduced once needed. Unused code ages and goes out of sync ;)

Reply:
rewrite the text
this will immediatly get an RPC interface, so I preferd wirign the hard parts, like the connectifivy first.